shared/settings/advanced/container.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import {TypedConnector} from '../../util/typed-connect'
+import {navigateAppend} from '../../actions/route-tree'
+import DBNuke from './index'
+
+const connector = new TypedConnector()
```

Comment:
I copied this from `settings/delete`. Do we prefer TypedConnector over the normal connect / mapStateToProps / mapDispatchToProps / mergeProps stuff?

Reply:
nope! this has actually been removed and is very bad for performance

Reply:
Fixed!